user-guide/Getting_started/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md

```diff
@@ -33,128 +33,41 @@ The DataMiner installer allows you to run a default DataMiner installation, whic
+> The installation requires that [Npcap](https://nmap.org/npcap/) or WinPcap (deprecated) is installed for systems intended for DataMiner Failover based on virtual IP. If you intend to configure a [Failover setup based on hostname](xref:Failover_configuration_in_Cube), this software will not be needed.
 
-### Default DataMiner installation
+## DataMiner installation
 
-1. Make sure you are logged into Windows with the correct user account:
-
-   - If you are using the DataMiner 10.2 installer or newer, use an account with administrator rights. During the installation, the user account will automatically be added to a local DataMiner user group named *Administrator (installer)*.
-   - If you are using a DataMiner installer older than version 10.2, use the server's local Administrator account. Do not use a regular user account with administrative rights.
+1. Make sure you are logged into Windows with the server's local Administrator account. Do not use a regular user account with administrative rights.
 
 1. Double-click the setup executable.
 
 1. Click *Install*.
 
-1. Enter the DataMiner ID.
-
-   > [!IMPORTANT]
-   > To get this DataMiner ID, contact <dataminer.licensing@skyline.be>. The DataMiner ID will uniquely identify the DataMiner Agent you are installing.
-   >
-   > If you are a Skyline employee, use the procedure from the [Skyline internal documentation](https://internaldocs.skyline.be/Corporate/OperatingProcedures/OP_RequestDecommissionMaintainInternalServer/Requesting_a_DMA_ID_or_a_DataMiner_license_for_an_internal_DataMiner_Agent.html) instead.
-
-1. Click next.
-
    The progress of the installation will be displayed. A *cancel* button in the lower right corner allows you to cancel the installation process if necessary.
 
-1. Once the installation is complete, click *next*.
-
-1. Click *go to Request.lic* to browse to *Request.lic*.
-
-1. Send the *Request.lic* file to <dataminer.licensing@skyline.be>, and wait until you receive the reply.
-
-   > [!TIP]
-   > See also: [Obtaining a DataMiner license](xref:DataminerLicenses)
-
-   > [!NOTE]
-   > If you are a Skyline employee, use the procedure from the [Skyline internal documentation](https://internaldocs.skyline.be/Corporate/OperatingProcedures/OP_RequestDecommissionMaintainInternalServer/Requesting_a_DMA_ID_or_a_DataMiner_license_for_an_internal_DataMiner_Agent.html) instead.
-
-1. Once you have received the license files, save these somewhere on the computer; however, not in the "Skyline DataMiner" folder.
+   Once the installation is complete, the configuration window will be displayed.
 
-1. In the *License* tab of the DataMiner Installer, click *browse and upload*, and navigate to the license files.
+1. Click *Start*.
 
-1. Once all files have been uploaded successfully, click *restart DataMiner*.
+1. Select the database type.
```

Comment:
Could you add info about each of the different options here, so users know what they are selecting?

Reply:
@MariekeGO 
This information is available in the PR about the pre-installed DataMiner Virtual Hard Disk as the configuration is the same there. 

user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_Virtual_Hard_Disk.md
## Configuring DataMiner

What would be the best approach to keep this info in both locations?

Reply:
I think it might be best to add this information fully in both locations; this both to keep the procedures user-friendly, and because these are two separate installers that may be updated independently, so that these options may not remain exactly the same for both in the future.